src/main/sdbench/sdbench_workload.cpp

```diff
@@ -876,7 +876,7 @@ static void AggregateQueryHelper(const std::vector<oid_t> &tuple_key_attrs,
   oid_t tuple_idx = 1;  // tuple2
   for (col_itr = 0; col_itr < column_count; col_itr++) {
     direct_map_list.push_back({col_itr, {tuple_idx, col_itr}});
-    col_itr++;
+//    col_itr++;
```

Comment:
This will trigger a warning in clang. Actually, here col_itr is added twice per each iteration, I am wondering if it's a typo?